### ISSUE TITLE
Checkpoint saving/ reloading info mismatch

### ISSUE THREAD

**Issue by Cli98** [zylo117/Yet-Another-EfficientDet-Pytorch#103]
Hi author @zylo117,

I am trying to reload checkpoints for my training model. But looks like there is a mismatch.

<img width="535" alt="image" src="https://user-images.githubusercontent.com/19630616/79415172-9f165500-7f7a-11ea-98b1-eeedf1b38fc2.png">

I'm trying to load checkpoint of epoch 10, step 90000, but after reloading it starts from epoch 5, step 90000. This makes me confusing. Which of them are correct?

Thank you,

**Comment by Cli98** [comment]
This happens during the time to pause and restart checkpoints.

**Comment by zylo117** [comment]
It happens when you change batchsize

**Comment by Cli98** [comment]
> It happens when you change batchsize

I changed nothing. Only checkpoint to reload. Check this.
<img width="649" alt="image" src="https://user-images.githubusercontent.com/19630616/79422079-2919ea00-7f8a-11ea-9a04-e0ec64f8e2e9.png">

Line 34200 and Line 34206. 

The reason to pause is due to the bug in training loss.

**Comment by zylo117** [comment]
then it could caused by a previous bug, since you didn't update my code until now.

run for a while and see if it will happen again.

the training and logging should not be affected, because they are recorded in steps

**Comment by Cli98** [comment]
I did not notice this. I have updated your code since yesterday. The bug you described for train loss. Do I need to update again?

**Comment by Cli98** [comment]
Nvm evaluation results look okay. Hopefully this is some occasional issues.
